### PR TITLE
Feat/chart repo pr action

### PR DESCRIPTION
**Pull request by deweya**
#### What is this PR About?
This is the GH Action that I shared on the container CoP call this morning. It was originally at https://github.com/deweya/chart-repo-pr-action, but I'd like to move it here since I think it will benefit CoP projects that want to publish their Helm charts but also want to maintain them separately from a central chart repository.

#### How do we test this?
I originally had a detailed workflow [in my personal repo](https://github.com/deweya/chart-repo-pr-action/blob/master/.github/workflows/test.yml). I'd like to carry this over too, but it **requires a personal access token** that can push to the fork of the desired Helm repository. This token must be configured as a secret (settings->secrets) so that test workflow has permission to push to the fork (without exposing the token, of course).

Can a maintainer of this repo create an access token with at least these permissions so that I can copy over my automated tests?
* public_repo
* read:discussion

Once created, can you add the access token as a secret to this repo?

#### Additional Details
My design requires users to create a fork of the target Helm chart repository. However, as I'm writing this, I realize that the chart repo might be under `redhat-cop`, along with other projects that might use this action. This makes creating a fork unreasonable, since you can't fork under the same org.

Is this correct to assume? Technically, with this action you don't need to fork, but the UX is bad since I wrote it with that requirement in mind. Let me know, and I can adjust accordingly.

cc: @redhat-cop/github-actions

### CODE REVIEW

File: chart-repo-pr-action/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+FROM alpine
```
Comment (sabre1041) [inline chart-repo-pr-action/Dockerfile:1]: Can an UBI image be used instead?

**Review by sabre1041 — CHANGES_REQUESTED** [review]
Comments added. Looks good, but need to rebase on RH UBI image

**Comment by deweya** [comment]
@sabre1041 Updated to use ubi8 minimal.

Do you have any thoughts around the questions I raised in the description (testing, viability of forks)?

File: chart-repo-pr-action/Dockerfile (hunk 1 above)
Comment (deweya) [reply chart-repo-pr-action/Dockerfile]: Updated

**Comment by deweya** [comment]
@sabre1041 Any updates on this PR?

**Comment by deweya** [comment]
@sabre1041 I updated the action to give users the choice to use a fork or just push directly to the chart repo.

`chart_repo` would be something like redhat-cop/helm-charts
`fork_owner` can be left blank to push directly to chart_repo, or something like "deweya" if I wanted to push to my fork first.

**Review by sabre1041 — APPROVED** [review]
LGTM

**Pull request merged**